includes/class-scheduler.php

```diff
@@ -22,6 +26,23 @@ public static function init() {
 		\add_action( 'activitypub_cleanup_followers', array( self::class, 'cleanup_followers' ) );
 
 		\add_action( 'admin_init', array( self::class, 'schedule_migration' ) );
+
+		// profile updates for blog options
+		if ( ! is_user_type_disabled( 'blog' ) ) {
+			\add_action( 'update_option_site_icon', array( self::class, 'blog_user_update' ) );
+			\add_action( 'update_option_blogdescription', array( self::class, 'blog_user_update' ) );
+			\add_action( 'update_option_blogname', array( self::class, 'blog_user_update' ) );
+			\add_filter( 'pre_set_theme_mod_custom_logo', array( self::class, 'blog_user_update' ) );
+			\add_filter( 'pre_set_theme_mod_header_image', array( self::class, 'blog_user_update' ) );
+		}
+
+		// profile updates for user options
+		if ( ! is_user_type_disabled( 'user' ) ) {
```

Comment:
This is the main reason I started from https://github.com/Automattic/wordpress-activitypub/pull/542

If needed I can rebase from master adding this myself